pokete_classes/ui_elements.py

```diff
@@ -118,6 +137,97 @@ def remove_c_obs(self):
         self.c_obs = []
 
 
+class BetterChooserItem(Box):
+    """Item for Better Choosebox"""
+
+    def __init__(self, height, width, text, ind):
+        super().__init__(height, width)
+        self.ind = ind
+        self.label = text
+        self.add_ob(self.label, 2, 1)
+
+    def choose(self):
+        """Rechars the frame to be highlighted"""
+        self.frame.rechar(corner_chars=["┏", "┓", "┗", "┛"],
+                         horizontal_chars=["━", "━"],
+                         vertical_chars=["┃", "┃"])
+
+    def unchoose(self):
+        """Rechars the frame to be not highlighted"""
+        self.frame.rechar(corner_chars=["┌", "┐", "└", "┘"],
+                         horizontal_chars=["─", "─"],
+                         vertical_chars=["│", "│"])
+
+
+class BetterChooseBox(Box):
+    """Better Choosebox using a tile layout"""
+
+    def __init__(self, columns, labels:[se.Text], name="", _map=None):
+        self.nest_label_obs = []
+        self.set_items(columns, labels, init=True)
+        super().__init__(3*len(self.nest_label_obs)+2,
+                         sum(i.width for i in self.nest_label_obs[0]) + 2,
+                         name, "q:close")
+        self.map = _map
+        self.__add_obs()
+        self.index = (0, 0)
+        self.get_item(*self.index).choose()
+
+    def set_index(self, _y, _x):
+        """Sets index and """
+        self.get_item(*self.index).unchoose()
+        self.index = (_y, _x)
+        self.get_item(*self.index).choose()
+
+    def get_item(self, _y, _x):
+        """Gives a choosen element"""
+        return self.nest_label_obs[_y][_x]
+
+    def input(self, _ev):
+        """Evaluates input input"""
+        _c = {"'w'": (-1, 0),
+             "'s'": (1, 0),
+             "'a'": (0, -1),
+             "'d'": (0, 1)}[_ev]
+        self.set_index((self.index[0] + _c[0])
+                            % len([i for i in self.nest_label_obs if len(i) >
+                                self.index[1]]),
+                       (self.index[1] + _c[1])
+                            % len(self.nest_label_obs[self.index[0]]))
+
+    def set_items(self, columns, labels:[se.Text], init=False):
+        """Sets the items shown in the box"""
+        for i in self.nest_label_obs:
+            for obj in i:
+                self.rem_ob(obj)
+        box_width = sorted(len(i.text) for i in labels)[-1]
+        label_obs = [BetterChooserItem(3, box_width + 4, label, i)
+                                       for i, label in enumerate(labels)]
+        self.nest_label_obs = [label_obs[i * columns:(i + 1) * (columns)]
```

Comment:
Unneeded parenthesis

```suggestion
        self.nest_label_obs = [label_obs[i * columns:(i + 1) * columns]
```